scripts/basic-workflow-tests.sh

```diff
@@ -236,13 +236,14 @@ test_train() {
         TRAIN_ARGS+=("--gguf-model-path" "${DATA_HOME}/instructlab/models/granite-7b-lab-Q4_K_M.gguf")
     fi
 
-    ilab model train "${TRAIN_ARGS[@]}"
+    # temporarily disabling this type of training until it is fixed
+    #  ilab model train "${TRAIN_ARGS[@]}"
 
     if [ "$FULLTRAIN" -eq 1 ]; then
-
+        DATA=$(find "${DATA_HOME}"/instructlab/datasets -name messages_*)
```

Comment:
Per the shellchecker
```suggestion
        DATA=$(find "${DATA_HOME}"/instructlab/datasets -name "messages_*")
```

Reply:
I think its a single quote? I literally pushed this as you reviewed lol

Reply:
if it's good with Shellchecker it's good with Nathan!